pkgs/misc/emulators/mednaffe/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, pkgconfig, gtk2, mednafen }:
+
+stdenv.mkDerivation rec {
+
+  name = "mednaffe-${meta.version}";
+
+  src = fetchurl {
+    url = "https://github.com/AmatCoder/mednaffe/archive/v${meta.version}.tar.gz";
+    sha256 = "1m7fmsnha342hvkgn7jhhs2pdz2p5yy2rsrw7ick0mib2xcvr2iq";
+  };
+
+  prePatch = ''
+    substituteInPlace src/mednaffe.c --replace "binpath = NULL" "binpath = \"${mednafen}/bin/mednafen\""
+  '';
+
+  buildInputs = with stdenv.lib; [ pkgconfig gtk2 mednafen ];
+
+  meta = with stdenv.lib; {
+    version = "0.8";
```

Comment:
taken care of. the mednafen package is using meta.version as well (it is where I saw it used).